data-final-fixes.lua

```diff
@@ -11,6 +11,11 @@ if mods["Krastorio2"] then
     data.raw["equipment-grid"]["spidertronmk3-grid"].equipment_categories = data.raw["equipment-grid"]["kr-spidertron-equipment-grid"].equipment_categories
     data.raw["equipment-grid"]["spidertron-builder-grid"].equipment_categories = data.raw["equipment-grid"]["kr-spidertron-equipment-grid"].equipment_categories
     data.raw["equipment-grid"]["immolator-grid"].equipment_categories = data.raw["equipment-grid"]["kr-spidertron-equipment-grid"].equipment_categories
+
+    if mods["space-exploration"] then
```

Comment:
You are trying to replace the default spidertron equipment grid width and height when it is using space-exploration and krastorio togheter, instead of this it might be better to be another option in the settings for the mod so that people can chose what they want instead of being forced with an option.

`
data.raw["equipment-grid"]["kr-spidertron-equipment-grid"].width = settings.startup["spidertron-g-width"].value
data.raw["equipment-grid"]["kr-spidertron-equipment-grid"].height = settings.startup["spidertron-h-height"].value
`

and in the settings.lua file
`
	{
		type = "int-setting",
		name = "spidertron-g-width",
		setting_type = "startup",
                minimum_value = 10,
                default_value = 10,
		order = "a111"
	},
	{
		type = "int-setting",
		name = "spidertron-h-height",
		setting_type = "startup",
                minimum_value = 6,
                default_value = 6,
		order = "a112"
	},
`
and in the locale/en/config.cfg
at [mod-setting-name]
`
spidertron-g-height=Vanilla Spidertron Grid Width
spidertron-h-height=Vanilla Spidertron Grid Height
`
at [mod-setting-description]
`
spidertron-g-height=Modify Vanilla Spidertron Grid Width
spidertron-h-height=Modify Vanilla Spidertron Grid Height
`

While at this we might also add it as a default option since it is defined:
in the prototypes/equipment_grid.lua
i think in the data:extend{ , i'm not 100% sure
`
data.raw["equipment-grid"]["spidertron-equipment-grid"].width = settings.startup["spidertron-g-width"].value
data.raw["equipment-grid"]["spidertron-equipment-grid"].height = settings.startup["spidertron-h-height"].value
`

Reply:
if you don't want to add these changes, i can merge the pull request as it is now, but i will do the extra changes.

Reply:
done